brainglobe_atlasapi/atlas_generation/atlas_scripts/australian_mouse_brain.py

```diff
@@ -0,0 +1,900 @@
+__version__ = "1"
+import multiprocessing as mp
+import os
+import tarfile
+import time
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import requests
+import SimpleITK as sitk
+from rich.progress import track
+
+from brainglobe_atlasapi.atlas_generation.mesh_utils import (
+    Region,
+    create_region_mesh,
+)
+from brainglobe_atlasapi.atlas_generation.wrapup import wrapup_atlas_from_data
+from brainglobe_atlasapi.structure_tree_util import get_structures_tree
+
+# Copy-paste this script into a new file and fill in the functions to package
+# your own atlas.
+
+### Metadata ###
+__version__ = 0
+ATLAS_NAME = "australian_mouse"
+CITATION = "lANKE et al. 2015, https://doi.org/10.1016/j.ymeth.2015.01.005"
+SPECIES = "Mus musculus"
+ATLAS_LINK = "https://imaging.org.au/AMBMC/"
+ORIENTATION = "iar"
+ROOT_ID = 9999
+RESOLUTION = 15
+BG_ROOT_DIR = Path.home() / "brainglobe_workingdir" / ATLAS_NAME
+DOWNLOAD_DIR_PATH = BG_ROOT_DIR / "downloads"
+
+ANNOTATION_URLS = {
+    "basalganglia": "https://osf.io/download/xgj4h",
+    "cerebellum": "https://osf.io/download/su2a9",
+    "cortex": "https://osf.io/download/hr39a",
+    "hippocampus": "https://osf.io/download/tjazr",
+    "diencephalon": "https://osf.io/download/9atc7",
+}
+REFERENCE_URL = "https://osf.io/download/g8p6a"
+REGION_IDS = {
+    "basalganglia": 1001,
+    "cerebellum": 1002,
+    "cortex": 1003,
+    "hippocampus": 1004,
+    "diencephalon": 1005,
+}
+PARALLEL = False
```

Comment:
I think we should remove the parallelisation here and where it's used below (see #428)

Reply:
removed